lib/curator.js

```diff
@@ -206,17 +220,24 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     '/tasks/credentials/request-credential-storage',
     relaxedAuthenticate, acceptableContent('urlencoded'),
     parseForm, parseJsonData, function(req, res, next) {
-    getDefaultViewVars(req, function(err, vars) {
-      if(err) {
-        return next(err);
+
+      if(req.body.credential[0]['@graph'].claim.id !== req.user.identity.id) {
```

Comment:
Add a TODO above this conditional that says we need to frame the credential first.